nautobot/core/templates/generic/object_retrieve.html

```diff
@@ -208,4 +208,33 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
     }
 
 </script>
+<script>
```

Comment:
maybe not needed to close one `script` and start a new `script` -- can this all be one `script` block?